tests/integration/dynamic_ip_test.py

```diff
@@ -979,3 +981,18 @@ def test_change_static_to_dhcp6_autoconf_with_disabled_ipv4(
     assert _poll(_has_ipv6_auto_gateway)
     assert _poll(_has_ipv6_auto_nameserver)
     assert _poll(_has_ipv6_auto_extra_route)
+
+
+@parametrize_ip_ver
+def test_activate_dummy_without_dhcp_timeout(ip_ver, dummy00):
+    ifstate = dummy00
+    if Interface.IPV4 in ip_ver:
+        ifstate[Interface.IPV4] = create_ipv4_state(enabled=True, dhcp=True)
+    if Interface.IPV6 in ip_ver:
+        ifstate[Interface.IPV6] = create_ipv6_state(
+            enabled=True, dhcp=True, autoconf=True
+        )
+    libnmstate.apply({Interface.KEY: [ifstate]})
+    time.sleep(NM_DHCP_TIMEOUT_DEFAULT + 1)
```

Comment:
Running a test for 45sec is a killer.
We need to mark it as slow and by default not run such tests unless explicitly asked.

Reply:
Create a pytest marker `slow` to stop it from running.
These test case can only be ran via `pytest --runslow` option.
The `runtest.sh --test-type integ_slow` will run the slow test case only.
The `runtest.sh --test-type integ` will not run slow test cases.

The travis will run the slow test case only in a separate metrix.

Reply:
Looks good to me.
Another option is to place slow tests under their own group/folder, then running them separately.
Unit are fast, Integration are slower and Slow are the slowest.
(seems much simpler to me)

But if you prefer the current option, I won't object.